index.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
-var objectAssign = require('object-assign');
-var sep = {};
+const sep = {};
 
-function create(input, opts) {
-	return input.map(function (line) {
+const create = (input, opts, menuLevel = 0) => {
+	return input.map(line => {
```

Comment:
I think it's more personal taste and we do it in all our packages. Consistency is key :).